### PR TITLE
Add demonstration for Bootstrap 5

### PR DESCRIPTION
**Pull request by gcko**
In conjunction with a [pull request](https://github.com/ghiscoding/Angular-Slickgrid/pull/657) against Angular-Slickgrid, adding Bootstrap 5 support to the Bootstrap theme.

- Refactored *.html files to correctly use Bootstrap (separated `<div class="row col-sm-12">...</div>` to `<div class="row"><div class="col-sm-12">...</div></div>`)
- Refactored the use of `form-inline` to the new Bootstrap 5 method of using [Inline Forms](https://getbootstrap.com/docs/5.0/forms/layout/#inline-forms)

### CODE REVIEW

File: bootstrap5-demo-with-translate/README.md
Comment (ghiscoding) [inline bootstrap5-demo-with-translate/README.md:1]: I guess that should be `Bootstrap 5` everywhere BS4 is mentioned right?

File: bootstrap5-demo-with-translate/.naverc
Comment (ghiscoding) [inline bootstrap5-demo-with-translate/.naverc:1]: what is a `.naverc` file? I've never seen this kinda of file before and google didn't tell me much about it. Do we really need this file?

File: bootstrap5-demo-with-translate/README.md
Comment (ghiscoding) [inline bootstrap5-demo-with-translate/README.md:7]: also make sure the folder name is correct, I assume it should be `cd bootstrap5-demo-with-translate` (which makes me think the BS4 folder name is incorrect)

**Review by ghiscoding — CHANGES_REQUESTED** [review]

File: bootstrap5-demo-with-translate/.naverc
Comment (gcko) [reply bootstrap5-demo-with-translate/.naverc:1]: i'll add it to `.gitignore`

File: bootstrap5-demo-with-translate/README.md
Comment (gcko) [reply bootstrap5-demo-with-translate/README.md]: Fixed in https://github.com/ghiscoding/angular-slickgrid-demos/pull/5/commits/14181c39cef77434ce50556d6dca53ab9feee981

File: bootstrap5-demo-with-translate/.naverc
Comment (gcko) [reply bootstrap5-demo-with-translate/.naverc]: Removed in https://github.com/ghiscoding/angular-slickgrid-demos/pull/5/commits/5735c8167240d49b17b93d3128cb1c0637e77e60 and added to `.gitignore` in https://github.com/ghiscoding/angular-slickgrid-demos/pull/5/commits/14181c39cef77434ce50556d6dca53ab9feee981

File: bootstrap5-demo-with-translate/README.md
Comment (gcko) [reply bootstrap5-demo-with-translate/README.md]: Fixed directory paths for all Bootstrap demos in https://github.com/ghiscoding/angular-slickgrid-demos/pull/5/commits/14181c39cef77434ce50556d6dca53ab9feee981, https://github.com/ghiscoding/angular-slickgrid-demos/pull/5/commits/ee92c60c6348191271c7a3589d0dbb7e2e7ce2e6, and https://github.com/ghiscoding/angular-slickgrid-demos/pull/5/commits/dd7f93d132c4ccbe75e2103998b57c69e548725f

**Review by ghiscoding — APPROVED** [review]

**Comment by ghiscoding** [comment]
Thanks, looks all good. I'll merge it later though since I think it would be better to wait for the main lib PR to be merged before merging this one.

**Pull request merged**
